eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +55,65 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
```

Comment:
This should be part of parent component, as this will contain multiple `AttendanceCard` components. You can check in `BookingsByDay` that inside these two wrappers there is a mapping function to render all attendance cards (we're only interested in what's inside that mapping function for this component).